chart/templates/role.yaml

```diff
@@ -1,204 +0,0 @@
-{{- if .Values.rbac.create }}
-{{- $env := .Values.env }}
-{{- if and ($env.WATCH_NAMESPACE) (ne $env.WATCH_NAMESPACE "default") }}
-# Split WATCH_NAMESPACE by commas and loop on them
-{{- $watchedNamespaces := (split "," $env.WATCH_NAMESPACE) -}}
-{{- range $watchedNamespaces }}
----
```

Comment:
@itamar-marom there is a conditional check, which is needed to scope the operator. Operator can watch multiple namespace, single namespace or all namespaces. On this either a role or clusterrole is created.

Reply:
OK, I understand. Fixed it but change it a little bit so there will be not extra role `druid-operator-manager-role-sc` - In the case of WATCH_NAMESPACES, the default ClusterRole will be created with only storageclass permissions.

Reply:
Generally, the role permissions are the same, also naming wise if you look at the chart, and the role.yaml under the `deploy` directory.
</br>
This is a bit tricky since it really depends on how the user deployed the operator. In the case of helm, it will be responsible to change the rbac to the correct one. In the case of regular deployment, the user should notice that the names are equal between the old and the new deployment.

Reply:
I detailed more in the migration docs. Eventually, it is important to check that the naming of the objects is the same in both deployments.